src/bitcoin-cli.cpp

```diff
@@ -359,6 +362,13 @@ class GetinfoRequestHandler: public BaseRequestHandler
             result.pushKV("balance", batch[ID_BALANCES]["result"]["mine"]["trusted"]);
         }
         result.pushKV("relayfee", batch[ID_NETWORKINFO]["result"]["relayfee"]);
+
+        result.pushKV("feerate_dist1", batch[ID_MEMPOOLINFO]["result"]["fee_histogram"]["fee_rate_groups"]["1"]["size"]);
+        result.pushKV("feerate_dist2", batch[ID_MEMPOOLINFO]["result"]["fee_histogram"]["fee_rate_groups"]["10"]["size"]);
+        result.pushKV("feerate_dist3", batch[ID_MEMPOOLINFO]["result"]["fee_histogram"]["fee_rate_groups"]["100"]["size"]);
+        result.pushKV("feerate_dist4", batch[ID_MEMPOOLINFO]["result"]["fee_histogram"]["fee_rate_groups"]["200"]["size"]);
+        result.pushKV("mempool_bytes", batch[ID_MEMPOOLINFO]["result"]["bytes"]);
```

Comment:
Yes. Because fee rate groups are:

1 (from 1 to 9)
10 (from 10 to 99)
100 (from 100 to 199)
200 (from 200 to 1000)

You can see them in the results for `bitcoin-cli getmempoolinfo "[1,10,100,200,1000]"`

Reply:
Ah got it! Tx with fee >= 1000 sat/vB will not be displayed.

Would it be better to update s/Above 200  |/200-1000   |/? IIUC `batch[ID_MEMPOOLINFO]["result"]["fee_histogram"]["fee_rate_groups"]["200"]["size"]` only considers fees from `200-1000`

```
./src/bitcoin-cli -signet getmempoolinfo "[1,10,100,200,1000]"
...
      "200": {
        "size": 0,
        "count": 0,
        "fees": 0,
        "from": 200,
        "to": 999
      },
...
```

Reply:
Possible. Or we can change 1000 to a higher number. Will see what other reviewers think and decide accordingly.

Reply:
Will keep this unchanged as nobody reviewed and had issues with it. Fee rates are normally less than 1000 sat/vB and we can ignore if someone uses higher in few transactions because this is just the fee rate distribution summary to help users and complete information.